test/fixtures/engagement.json

```diff
@@ -0,0 +1,15 @@
+{
+  "engagements": [
```

Comment:
can we call this marketing_engagement, because there can be different attributes for other engagemenrs such as content engagements